### PR TITLE
Changes to the Landing page

### PR DESCRIPTION
**Pull request by nglglhtr**
##### Description

  * remove "Developers" &  "Network"
  * add 2 new descriptions with buttons to "What is Gitcoin" sec.
  * move newsletter section to 3rd screen

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/8171
##### Testing

Tested locally

### Desktop

image
image
image

### Mobile
Screenshot 2021-01-08 at 6 51 01 PM
Screenshot 2021-01-08 at 6 51 14 PM
Screenshot 2021-01-08 at 6 51 20 PM
Screenshot 2021-01-08 at 6 51 26 PM

### Video
https://www.icloud.com/iclouddrive/04Y5aseNyERZFCk0yJ7XuY1IQ#Screen_Recording_2021-01-08_at_6.59.52_PM

### CODE REVIEW

**Review by aamustapha — APPROVED** [review]
Good work

File: app/assets/landingpage/app.css [hunk 1]
```diff
@@ -133,980 +112,763 @@ a.btn, button.btn {
...
-  }
-}
+    margin-right: 1.5em; } }
+
 section#footerconsensys figure {
   width: 200px;
   margin: auto;
-  padding: 3em 0em;
-}
+  padding: 3em 0em; }
```
Comment (thelostone-mc) [inline app/assets/landingpage/app.css:874]: ^ could you undo all these indent changes

File: app/assets/landingpage/section_whatisgitcoin.scss [hunk 2]
```diff
@@ -19,18 +19,21 @@ section#whatisgitcoin{
...
+      a {
+        margin-right: 26px;
+      }
 		}
 
-	}
+  }
 
 
 }
```
Comment (thelostone-mc) [inline app/assets/landingpage/section_whatisgitcoin.scss:39]: could you reindent this file with 1 tab = 2 spaces (just like the other files)

File: app/retail/templates/home/index2020.html [hunk 3]
```diff
@@ -84,34 +84,21 @@
...
-      Join the Community of Web3’s Top Developers.
-
+      Join the open source community of Web3’s top developers building the best decentralized applications.
     </h1>
-
-
-    <p>Learn the latest tech, earn money, and network with top developers passionate about building and sustaining a remote-first open source community.</p>
-
-    <a class="btn icon fe-pulse" href="{% if not github_handle %}/login/github/?next=products{% else %}/townsquare{% endif %}">
-      <img src="/static/v2/images/helmet-white.svg">
```
Comment (thelostone-mc) [inline app/retail/templates/home/index2020.html:110]: If this is not used anywhere -> can we remove the image as well ?

File: app/retail/templates/home/index2020.html [hunk 4]
```diff
@@ -124,40 +111,56 @@ <h1>
...
+      <div>
+      <input type="email" name="EMAIL" placeholder="Enter your email" required>
 
+      <button type="submit" class="btn fe-pulse">Subscribe</button>
+    </div>
 
 
+  </form>
 
+</section>
```
Comment (thelostone-mc) [inline app/retail/templates/home/index2020.html:163]: ^ indent fix needed

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
Left a few comments!
Could you also add a screenshot and ensure it also works for mobile display

**Comment by PixelantDesign** [comment]
@nglglhtr could you post a video?

**Comment by nglglhtr** [comment]
adding the copy to be added here: https://docs.google.com/document/d/1cDmUr80AbL1HfYLLou8lvIehVjGt-hw9ddXK9uueAkU/edit#

**Comment by nglglhtr** [comment]
> @nglglhtr could you post a video?

here's the link to a desktop scroll video: https://www.icloud.com/iclouddrive/04Y5aseNyERZFCk0yJ7XuY1IQ#Screen_Recording_2021-01-08_at_6.59.52_PM

**Comment by frankchen07** [comment]
Minor changes:

"Get grant funding" -> can we keep the capitalization consistent? -> "Get Grant Funding" 

Does "Enter Gitcoin" just put them back on Townsquare? I would recommend removing the ability to just "enter" and end up on Townsquare because we've received plenty of feedback on how folks are confused when they get there. The onboarding idea for the landing page was to direct them down one specific path to reduce the problem of confusion.

Screen Shot 2021-01-08 at 13 39 05

Is there any way to help this text stand out, or feel more like "points" rather than just text thrown on there? It's a bit hard to read:

Screen Shot 2021-01-08 at 13 38 22

> up skill

I think it's supposed to be one word "upskill".

> ...the ecosystem, Get involved...

Should be a period. "...the ecosystem. Get involved..."

The rest looks pretty good. Just from watching the video though, I still feel like the zoom is pretty strong. I would want to see more with less scrolling, but pp to design though cc @PixelantDesign

**Review by chibie — APPROVED** [review]

**Comment by nglglhtr** [comment]
can we merge this?

**Comment by frankchen07** [comment]
I had a comment with a bunch of changes @nglglhtr, did they make it in?

**Comment by nglglhtr** [comment]
> Does "Enter Gitcoin" just put them back on Townsquare? I would recommend removing the ability to just "enter" and end up on Townsquare because we've received plenty of feedback on how folks are confused when they get there. The onboarding idea for the landing page was to direct them down one specific path to reduce the problem of confusion.

Yes it does - can remove the button altogether for now

**Comment by nglglhtr** [comment]
Fixed the heights of the buildings in the background to create more space for the text, here's an 8x video: 

https://user-images.githubusercontent.com/10832414/104443599-a6779700-55bc-11eb-91c0-7fed804b599f.mp4

**Comment by frankchen07** [comment]
@nglglhtr, the all caps for the actions look a little funny to me, and I feel like it crowds the space a little bit. 

Can we just simplify and remove all of the sub-text (START an open source project. LEARN ...etc.), and leave the larger title text there?

So just keep:

Screen Shot 2021-01-13 at 13 26 38

Everything else looks good to go! Thanks for helping out with these changes! :)

**Comment by thelostone-mc** [comment]
@PixelantDesign  + @frankchen07 ddo hit approve once the PR Is good from your side and I can get it in after that

**Comment by nglglhtr** [comment]
> @nglglhtr, the all caps for the actions look a little funny to me, and I feel like it crowds the space a little bit.
> 
> Can we just simplify and remove all of the sub-text (START an open source project. LEARN ...etc.), and leave the larger title text there?

Yes, for sure, not a problem; I do not really have a strong opinion for or against a design change - my UI skillz are pretty much nonexistent lol; relying on yours & @PixelantDesign's judgement here :)

**Comment by nglglhtr** [comment]
Updated screenshot:
image

**Comment by frankchen07** [comment]
I read the copy a few times to make sure that even though we remove the sub-text we're still conveying 90+% using the title, larger text. No worries, text is easy to change if we decide to do so with any new rebranding copy work.

**Review by frankchen07 — APPROVED** [review]
LGTM!

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**